src/components/accessories/feedback/utils.ts

```diff
@@ -0,0 +1,13 @@
+import * as params from "../../../customization/feedback.json";
+import { FeedbackParams } from "./type";
+
+const collectorUrl =
+  "https://openhospital.atlassian.net/s/d41d8cd98f00b204e9800998ecf8427e-T/r5gghz/b/3/bc54840da492f9ca037209037ef0522a/_/download/batch/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector.js?locale=en-US&collectorId=53f7755c";
+
+export const FEEDBACKPARAMS: FeedbackParams = {
```

Comment:
The pattern that we're gonna use for variables is camelCase (unless they are used as constant key inside a const.ts file that would be uppercase). So please rename that var in "feedbackParams".

Reply:
OK !